testsuite/features/secondary/allcli_reboot.feature

```diff
@@ -37,19 +37,16 @@ Feature: Reboot systems managed by SUSE Manager
     And I run "rhn_check -vvv" on "sle_client"
     Then I wait and check that "sle_client" has rebooted
 
-# WORKAROUND
-#    This scenario is disabled as long as we have failures during CentOS reboots
-#
-#@centos_minion
-#  Scenario: Reboot the CentOS minion and wait until reboot is completed
-#    Given I am on the Systems overview page of this "ceos_ssh_minion"
-#    When I follow first "Schedule System Reboot"
-#    Then I should see a "System Reboot Confirmation" text
-#    And I should see a "Reboot system" button
-#    When I click on "Reboot system"
-#    Then I should see a "Reboot scheduled for system" text
-#    When I wait at most 600 seconds until event "System reboot scheduled by admin" is completed
-#    Then I should see a "Reboot completed." text
+@centos_minion
+  Scenario: Reboot the CentOS minion and wait until reboot is completed
+    Given I am on the Systems overview page of this "ceos_ssh_minion"
```

Comment:
centos or centos ssh in the title scenario?

Reply:
Our centos minion, like the ubuntu minion, is always accessed via SSH

Reply:
After verification, you are right and I am wrong. I will fix it.

Reply:
Should we then reflect that in the title?

Reply:
I just said I will fix it :troll:

Reply:
Done

Reply:
haha sorry, GitHub doesn't seem to update fast enough :trollface: